app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaticPagesController < ApplicationController
+  def home
```

Comment:
```suggestion
  def home; end
```
tương tự các method ở dưới nha. nếu bên trong k có sử lý gì.